packages/vitest/rollup.config.js

```diff
@@ -57,6 +57,8 @@ const external = [
   'node:worker_threads',
   'node:fs',
   'rollup',
+  'rollup/parseAst',
```

Comment:
We shouldn't import from rollup because it's not in our dependencies. We shouldn't add it to our dependencies because we might have a version mismatch with Vite.

We might be able to trick Vite into giving us `parse` method from `transform` context somehow?

```ts
let _parse
export function ParseStorePlugin() {
  return { name: 'vitest:parse-store', transform() { _parse ??= this.parse } }
}
export function getParser() {
  return _parse
}
```

Reply:
This way seems a little strange. But I'll give it a try.

Reply:
I have seen some discussion related to `parseAst` in https://github.com/vitejs/vite/pull/14508#discussion_r1342340771. Looks like we should use `parseAst`.

Reply:
Yeah, `parseAst` is exposed on `server.pluginContext.parse` - we can use it from there without importing `rollup/parseAst`

Reply:
Oh, the context is not exposed there, let me look into it

Reply:
> I have seen some discussion related to parseAst 

This discussion doesn't matter for us because we cannot have `rollup` in dependencies (see the first comment).

Reply:
Yes, I agree